plugins/registry/etcd/etcd.go

```diff
@@ -36,10 +37,14 @@ type etcdRegistry struct {
 
 func NewRegistry(opts ...registry.Option) registry.Registry {
 	e := &etcdRegistry{
-		options:  registry.Options{},
+		options:  registry.Options{Addrs: []string{os.Getenv("MICRO_REGISTRY_ADDRESS")}},
```

Comment:
if the env var is blank this will create a blank entry, regardless of whether its parsed later, that's not useful. Please remove.

Reply:
I've removed it, it will be added into registry options as default address instead of 127.0.0.1:2379 when it's not blank.

Reply:
BTW, the unit test passed in my local environment, shouldn't be an issue caused by my codes.